### PR TITLE
Replace submodules with pip install

### PR DESCRIPTION
**Pull request by matthiasdiener**

### CODE REVIEW

**Comment by inducer** [comment]
How come this has the uninstall branch in it?

**Comment by matthiasdiener** [comment]
> How come this has the uninstall branch in it?

It's for my testing (to check whether install/uninstall with various options keeps on working with these changes). Please ignore this PR for now.

**Comment by matthiasdiener** [comment]
Ready for review.

File: install-conda-dependencies.sh [hunk 1]
```diff
@@ -6,4 +6,4 @@ echo "==== Installing Conda packages for $(which conda)"
 
 conda info --envs
 
-conda install --yes pocl numpy pyvisfile pyopencl islpy
```
Comment (inducer) [inline install-conda-dependencies.sh:9]: PyOpenCL and islpy take a bit of time to build, and I expect them to change infrequently. So there's a trade-off of install speed vs. hacking convenience here. I'm OK with leaving them source-built as you have here, since it's just a few minutes of build time, but I figured I would bring up that the option exists.

File: install.sh [hunk 2]
```diff
@@ -72,5 +72,5 @@ echo "# Emirge is now installed. Please run the following commands            #"
 echo "# to test the installation (assuming your shell is bash):               #"
 echo "# $ source $conda_prefix/bin/activate dgfem                             #"
 echo "# $ export OCL_ICD_VENDORS=$MY_CONDA_DIR/envs/dgfem/etc/OpenCL/vendors  #"
```
Comment (inducer) [inline install.sh:74]: Why is that needed? The ocl-icd (Linux)/khronos loader (MacOS) in Conda should default to this.

File: install.sh [hunk 3]
```diff
@@ -72,5 +72,5 @@ echo "# Emirge is now installed. Please run the following commands            #"
...
-echo "# $ python mirgecom/examples/wave-eager.py                              #"
+echo "# $ python /path/to/mirgecom/examples/wave-eager.py                     #"
 echo "#########################################################################"
```
Comment (inducer) [inline install.sh:76]: This set of instructions is a nice touch, but the box will typically come out misaligned:
```
#########################################################################
# Emirge is now installed. Please run the following commands            #
# to test the installation (assuming your shell is bash):               #
# $ source /home/andreask_work/miniforge3/bin/activate dgfem                             #
# $ export OCL_ICD_VENDORS=/home/andreask_work/miniforge3/envs/dgfem/etc/OpenCL/vendors  #
# $ python /path/to/mirgecom/examples/wave-eager.py                     #
#########################################################################
```
How about not having a boundary on the right?

File: install.sh [hunk 4]
```diff
@@ -72,5 +72,5 @@ echo "# Emirge is now installed. Please run the following commands            #"
 echo "# to test the installation (assuming your shell is bash):               #"
 echo "# $ source $conda_prefix/bin/activate dgfem                             #"
 echo "# $ export OCL_ICD_VENDORS=$MY_CONDA_DIR/envs/dgfem/etc/OpenCL/vendors  #"
-echo "# $ python mirgecom/examples/wave-eager.py                              #"
+echo "# $ python /path/to/mirgecom/examples/wave-eager.py                     #"
```
Comment (inducer) [inline install.sh:75]: This didn't end up cloning/installing mirgecom (AFAICT). I think it should.

Comment (matthiasdiener) [reply install.sh:75]: Yes, this currently doesn't clone mirgecom. I can add that before cloning the other repos; in this case, we wouldn't even need to `wget` the requirements.txt (but just use the one from the mirgecom clone).

Comment (inducer) [reply install.sh:75]: I'd say go for it.

File: install.sh (hunk 2 above)
Comment (matthiasdiener) [reply install.sh:74]: Seems to be unnecessary, will remove.

File: install.sh (hunk 3 above)
Comment (matthiasdiener) [reply install.sh]: Fixed.

File: install.sh (hunk 4 above)
Comment (matthiasdiener) [reply install.sh]: Done.

**Comment by matthiasdiener** [comment]
Should be ready for another round of reviews.

**Comment by inducer** [comment]
Thanks for working on this!
